packages/babel-generator/package.json

```diff
@@ -26,6 +26,7 @@
   "devDependencies": {
     "@babel/helper-fixtures": "workspace:^",
     "@babel/parser": "workspace:^",
+    "@jridgewell/trace-mapping": "^0.3.4",
     "@types/jsesc": "^2.5.0",
     "@types/source-map": "^0.5.0",
```

Comment:
ditto.

Reply:
`generator` does, because it uses `SourceMapGenerator`. `trace-mapping` only offers a Consumer API.